sql/src/test/resources/model/model_file_csv.json

```diff
@@ -0,0 +1,27 @@
+{
+  version: '1.0',
```

Comment:
Since Calcite supports strict json, I think we should change this to use strict json so that github will not flag these.